guides/v2.1/comp-mgr/upgrader/upgrade-start.md

```diff
@@ -11,25 +11,24 @@ functional_areas:
   - Upgrade
 ---
 
-<h2 id="compman-prereq">Prerequisites</h2>
-Before continuing, complete all tasks discussed in <a href="{{ page.baseurl }}/comp-mgr/prereq/prereq_compman.html">Prerequisites</a>.
+## Prerequisites {#compman-prereq}
```

Comment:
Please complete these changes in the v2.2 version if needed, as its not symlinked.